### PR TITLE
feat(LayerEntry): Convert from MUI to bootstrap

### PR DESCRIPTION
**Pull request by labdala**
In this version we have place holders for the Tooltips.
We also change the style of the bar to look more closely to the original App. 

**To-do**: optimize re-renders

### CODE REVIEW

**Comment by labdala** [comment]
There is something up with the layers that make them re-render more often than they should. That is aside from the `applyContrastSensitiveStyleToElement` to create the dark mode.

**Comment by PaulHax** [comment]
Here is another test case that loads a "label image" layer.  Useful because the extra layer changes the layout.  It uses test data from the main repo and loads the data via a local HTTP server.  We can go over how to run this later:

```
    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/itk-vtk-viewer@11.12.4/dist/itkVtkViewer.js"></script>
    <script>
      const container = document.querySelector('.content')
      const ipfsImage = new URL("http://localhost:8082/test-data/HeadMRVolume.nrrd")
      const label = new URL("http://localhost:8082/test-data/HeadMRVolumeLabels.nrrd")
      const uiMachineOptions = { href: new URL("/src/materialUIMachineOptions.js", document.location.origin).href }
      itkVtkViewer.createViewer(container,
        {
        image: ipfsImage,
        labelImage: label,
        rotate: false,
        config: { uiMachineOptions },
        })
    </script>
```

**Comment by labdala** [comment]
Opened issue #28 to address problem pointed out in commits [47a78b6](https://github.com/InsightSoftwareConsortium/itk-viewer-bootstrap-ui/pull/17/commits/47a78b675766835bf9e4beb9b0f6bf94dceb44f1) and [4cab70d](https://github.com/InsightSoftwareConsortium/itk-viewer-bootstrap-ui/pull/17/commits/4cab70da41d0ebb15495b77a4a43bc686605a3e3): 

the TransferFunctionWidget should disappear when using only labelImage.

**Review by PaulHax — CHANGES_REQUESTED** [review]
Can we have Image and LabelImage on the same row to save some vertical pixels?  Main repo app look:
image

The picture and label icons on the right side seem clickable with that focus issue we had on the "sample distance" icons.

File: src/Images/ImagesInterface.jsx [hunk 1]
```diff
@@ -39,18 +39,29 @@ function ImagesInterface(props) {
...
 
+  const isLabelOnly = () => {
+    const type = layersContext.type
+    if (type === 'labelImage') {
+      return true
+    }
+    return false
+  }
+
+  console.log(layersContext)
```
[the remark this answers is not in the document]
Comment (labdala) [reply src/Images/ImagesInterface.jsx:50]: The disadvantage of saving vertical space is that, for files that have long names, the visibility button does not fit the screen:
image
I am not sure if an user would load more than 2 files, but I suppose they would run into the same issue(?)

[the remark this answers is not in the document]
Comment (PaulHax) [reply src/Images/ImagesInterface.jsx]: Good point.  Rows per image it is then.

[the remark this answers is not in the document]
Comment (labdala) [reply src/Images/ImagesInterface.jsx]: yay! Do you think this PR is ready to merge?

**Review by PaulHax — APPROVED** [review]
Looks good!

**Pull request merged**